### ISSUE TITLE
Myra doesn't work with MonoGame 3.7

### ISSUE THREAD

**Issue by m039** [MyraUI/Myra#34]
I tried a very simple project with Myra and MonoGame and always encounter TargetInvocationException. I used [Tutorial Formatted Text](https://github.com/rds1983/Myra/wiki/Tutorial-Formatted-Text) for this purpose. The exception raises here:
```
_formattedText = new FormattedText
  {
  	Font = DefaultAssets.Font,
  	Text = "Lorem ipsum",
  	Width = 500
  };
```

I also found that this happens in BitmapFontHelper class at:
```
var result = (SpriteFont) constructorInfo.Invoke(new object[]
 {
  textureRegion.Texture, glyphBounds, cropping,
  chars, data.LineHeight, 0, kerning, ' '
 });
```

The error message: `"{System.InvalidOperationException: Invalid SpriteFont. Character map must be in ascending order.   at Microsoft.Xna.Framework.Graphics.SpriteFont..ctor (Microsoft.Xna.Framework.Graphics.Texture2D texture, System.Collections.Generic.List1[T] glyphBounds, System…}"`.

When I downgraded to MonoGame 3.6, the exception had gone. The project is tested on Mac.

**Comment by nkast** [comment]
See [6125](https://github.com/MonoGame/MonoGame/pull/6125 ) and [5874](https://github.com/MonoGame/MonoGame/pull/5874).

**Comment by rds1983** [comment]
Thanks for reporting! This issue had been fixed. Fix would be included in the next release (0.7.0)
